helpers.py

```diff
@@ -0,0 +1,89 @@
+import os
+import torch
+import shutil
+from faster_whisper import WhisperModel
+
+
+def vocal_extraction(audio_path, extract=True):
+    if extract:
+        os.system(
+            f'python3 -m demucs.separate -n htdemucs --two-stems=vocals "{audio_path}" -o "temp_outputs"'
+        )
+        vocal_target = os.path.join(
+            "temp_outputs",
+            "htdemucs",
+            os.path.splitext(os.path.basename(audio_path))[0],
+            "vocals.wav",
+        )
+    return vocal_target
+
+
+# READ, MAP TIMESTAMPS AND REFORMAT FOR READIBILITY
+def text_speaker_map(whisper_results, temp_path):  # add whisper results here?
+    spk_tp = []
+    with open(os.path.join(temp_path, "pred_rttms", "mono_file.rttm"), "r") as f:
+        lines = f.readlines()
+        for line in lines:
+            line_list = line.split(" ")
+            s = int(float(line_list[5]) * 1000)
+            e = s + int(float(line_list[8]) * 1000)
+            spk_tp.append([s, e, int(line_list[11].split("_")[-1])])
+
+    with open("speaker_transcript.txt", "w", encoding="utf-8-sig") as f:
+        for i, seg in enumerate(whisper_results):
+            start = seg["start"]
+            end = seg["end"]
+            text = seg["text"]
+            spk = spk_tp[i][2]
+            # s = 'Speaker_'+str(spk), start, "-", str(end)+":", text
+            # print('Speaker_'+str(spk), start, "-", str(end)+":", text)
+            f.write(
+                "Speaker_"
+                + str(spk)
+                + " "
+                + str(start)
+                + "-"
+                + str(end)
+                + ":"
+                + str(text)
+                + "\n"
+            )
+    return spk_tp
```

Comment:
For readibilty, use f-strings! 
In this case:
f"Speaker_{spk} {start}-{end}:{text}\n"